content/docs/setup/kubernetes/getting-started/index.md

```diff
@@ -1,30 +1,25 @@
 ---
 title: Getting Started
-description: Download and install Istio.
+description: Download, install, and try out Istio.
 weight: 5
 skip_toc: true
-keywords: [kubernetes]
+keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-If you’re new to Istio and just want to try it out, the quickest way to get started is
-by downloading and installing Istio's built-in **demo** configuration.
-Istio can also be installed and customized in many other ways, depending on your platform and intended use.
-Check out our [general installation instructions](/docs/setup/) for details.
-
-To get started, proceed as follows:
+This guide lets you try out Istio quickly and it's the ideal starting
+point if you are new to the project. To get started, download and install
+Istio's built-in **demo** configuration:
 
 1. [Setup your platform](/docs/setup/kubernetes/platform-setup/)
 1. [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release)
 1. [Follow the quick-start installation instructions](/docs/setup/kubernetes/install/kubernetes)
 
-Once you've installed Istio, a good way to start evaluating Istio's features is using our
-[Bookinfo sample application](/docs/examples/bookinfo/).
-You can use this sample to experiment with Istio’s features for traffic routing,
-fault injection, rate limiting, etc..
+With Istio installed, you can [deploy the Bookinfo sample](/docs/examples/bookinfo/#if-you-are-running-on-kubernetes)
+and evaluate Istio's features for traffic routing, fault injection, rate
+limiting, etc. Explore the various [Istio tasks](/docs/tasks/), and try out the
+tasks that interest you. If you are not sure where to begin, the following
+tasks are a good place to start:
 
-1. [Deploy the Bookinfo sample](/docs/examples/bookinfo/#if-you-are-running-on-kubernetes)
-1. Explore various [Istio tasks](/docs/tasks/), depending on your interest.
-    The following tasks are a good start for beginners:
     * [Request routing](/docs/tasks/traffic-management/request-routing/)
```

Comment:
These list items are not formatting properly. Need to unindent and should change `*` to `-`.

Reply:
@frankbu Fixed, thanks.